src/raft.c

```diff
@@ -24,6 +24,10 @@
 #define DEFAULT_MAX_CATCH_UP_ROUNDS 10
 #define DEFAULT_MAX_CATCH_UP_ROUND_DURATION (5 * 1000)
 
+static int raft_io_fsm_compat(struct raft *r,
```

Comment:
Nit: should be camel-case (e.g. `ioFsmCompat` or `raftIoFsmCompat`), since it's a private function.